app/battle/knight.py

```diff
@@ -0,0 +1,44 @@
+class Knight:
+    def __init__(self,

```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability